zqd/space/endpoint.go

```diff
@@ -82,13 +84,28 @@ func spaceInfo(spaceName, path string) (*api.SpaceInfo, error) {
 	return s, nil
 }
 
+func parseSpace(path string) (string, string) {
+	//XXX need to sanitize this path
+	spaceName := strings.Replace(path, "/space/", "", 1)
+	if spaceName == "" {
+		return "", ""
+	}
+	if strings.HasSuffix(spaceName, "/packet") {
+		return "", spaceName[:len(spaceName)-7]
+	}
+	return spaceName, ""
+}
+
 func HandleInfo(w http.ResponseWriter, r *http.Request) {
 	if r.Method != http.MethodGet {
 		http.Error(w, "bad method", http.StatusBadRequest)
 		return
 	}
-	//XXX need to sanitize spaceName
-	spaceName := strings.Replace(r.URL.Path, "/space/", "", 1)
+	spaceName, pcapSpace := parseSpace(r.URL.Path)
+	if pcapSpace != "" {
+		pcap.HandleGet(w, r, pcapSpace)
+		return
+	}
```

Comment:
This is kind of strange... the packet search endpoint is bolted into the `/space/info` endpoint? Why not just create this as a separate endpoint?

Reply:
Yeah, we need a router.  It's "demo quality" for now...